cinema/src/main/java/com/greg/cinema/entity/TicketTypeEntity.java

```diff
@@ -0,0 +1,61 @@
+package com.greg.cinema.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "ticket_types")
+public class TicketTypeEntity {
+	
+	@Id
+	@GeneratedValue
+	private Integer id;
+	
+	@Column
+	private String name;
+	
+	@Column
+	private float price;
```

Comment:
dlaczego float to bardzo zły typ dla $$?

Reply:
Racja, zwykle używam typów całkowitych do przechowywania najmniejszej jednostki monetarnej np. grosze.  Ale w tym projekcie uznałem, że nie ma wykonywania skomplikowanych operacji na pieniądzach i dla prostoty użyłem float